### DISCUSSION TITLE
Problem Adding Second Email to my Account

### DISCUSSION THREAD

**Discussion by mdebellis** [community/community#133568] [Code Security] [answered]
### Select Topic Area

Question

### Body

I have the following message in my email settings: "You have a single verified email associated with your GitHub account. Add an additional verified email address in case you lose access to your primary email." I added a second email and it sent a verification request to that email. It asked me to login to Github. I didn't think I had a Github account set up for that email so I went to create one but it said there WAS an account for the second email. I tried logging in hoping it would remember my password but it didn't. So I did "Reset Password" and it said that there is no Github account associated with that email.

**Comment by mdebellis** [comment, marked as answer]
I figured out the problem. It wanted me to login with my main account to verify the second account. Did that and it worked.

**Reply by ersurajkumar1999** [reply]
> I figured out the problem. It wanted me to login with my main account to verify the second account. Did that and it worked.

**You have a single verified email associated with your GitHub account. Add an additional verified email address in case you lose access to your primary email.**

**Comment by ersurajkumar1999** [comment]
remote: error: GH007: Your push would publish a private email address.
remote: You can make your email public or disable this protection by visiting:     
remote: http://github.com/settings/emails
